### PR TITLE
Scott/all payments

### LINKED ISSUE #172: Add Transaction for Ledger Endpoint to CLI
### Expected Behavior
```
stellar ledger transactions <ID>
```
The ID is the Ledger's ID
This should also take cursor, order, and page-limit as arguments.

### Transaction details to display
```
id
source_account
created_at
```
https://www.stellar.org/developers/horizon/reference/endpoints/transactions-for-ledger.html

### PR DESCRIPTION
**Pull request by correlator**
Added all payments endpoint to the CLI.  As @robertDurst pointed out, source amount is not actually returned as a part of the path_payment operation from horizon so I updated that resource and corresponding models.

Also had some time on a plane so added transactions for ledger to the CLI as well.

Resolves #196  Resolves #172 

### Is there a GIF that reflects how this work made you feel?
Alt Text

### CODE REVIEW

**Comment by robertDurst** [comment]
This is great -- get rid of the redundancies!

I was curious, is there any way to test this? Ie is there a way to catch the error I found without actually manually running the CLI?

**Comment by correlator** [comment]
Well, there are 2 places where we test this.  The first is in the fixture file that contains our best understanding of what we expect horizon to return https://github.com/kbacha/stellar-sdk/blob/master/client/fixtures/operations/path_payment.json . You can see that is now updated in this PR.  The other place is in the endpoints testing suite.  Problem is I'm not sure how to specifically request a path_payment from the horizon api.

**Comment by robertDurst** [comment]
@correlator that is valid. I believe you'd have to find one in the ledger and then request it specifically -- thus to guarantee all tests are consistent in the future, we'd want to use the public network.

File: cli/src/fmt/simple/payment.rs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                append_to_buffer!(buf, "Account:          {}", create_account.account());
+                append_to_buffer!(buf, "Funder:           {}", create_account.funder());
+                append_to_buffer!(
+                    buf,
+                    "Starting Balance: {}",
+                    create_account.starting_balance()
+                );
+            }
+            &OperationKind::Payment(ref payment) => {
+                append_to_buffer!(buf, "Operation Kind: Payment");
```
Comment (kbacha) [inline cli/src/fmt/simple/payment.rs]: Feels like we could have the "kind" put first

File: cli/src/fmt/simple/payment.rs [hunk 2]
```diff
@@ -0,0 +1,73 @@
+use fmt::Render;
+use super::Simple;
+use stellar_client::resources::{Operation, OperationKind};
+
+impl Render<Operation> for Simple {
+    fn render(&self, operation: &Operation) -> Option<String> {
+        let mut buf = String::new();
+        append_to_buffer!(buf, "ID:             {}", operation.id());
+        match operation.kind() {
```
Comment (kbacha) [inline cli/src/fmt/simple/payment.rs]: Clippy would say to match by the deref.

**Pull request merged**